util/topgen/lib.py

```diff
@@ -880,6 +880,15 @@ def find_module(
     return mods[0] if mods else None
 
 
+def find_module_by_name(modules: List[Dict[str, object]],
+                        name: str) -> Optional[Dict[str, object]]:
+    '''Return the (first) module with a given name, or None.'''
```

Comment:
Nit. Although we don't check or enforce anything, and have much code which does the opposite, triple double-quotes for doc comments is normally the standard.